docs/docusaurus.config.js

```diff
@@ -59,6 +59,13 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      algolia: {
+        appId: '3SZT2K1T21',
+        // Public API key: it is safe to commit it
+        apiKey: '959618f671e39b8520016368d09232d2',
```

Comment:
why is this key needed?
where is the search service running?

Reply:
Algolia is a 3rd party service that crawls your website and provides an index. Many docusaurus based sites use it to provide search because it has very good/easy integration with it.

Reply:
Where is the index stored?
What do they get out of it?

Reply:
Why do we need a crawler if we already have all the data?